pubsublite-beam-io/README.md

```diff
@@ -58,8 +58,12 @@
                 .setName(SubscriptionName.of("my-sub"))
                 .build())
             .setFlowControlSettings(FlowControlSettings.builder()
-                .setBytesOutstanding(100_000_000)  // 100 MB
+                .setBytesOutstanding(50_000_000)  // 50 MB
                 .setMessagesOutstanding(Long.MAX_VALUE)
                 .build())
             .build()));
     ```
+
+1. If operating in a memory-constrained environment, making your pipeline
+options inherit from `PubsubLitePipelineOptions` will put a 1GiB per
```

Comment:
This isn't the normal way to use options.  Just make the limit unset by default and let users set the limit if they want.  If you want to have a default value that's easy to use, have two options, one bool that turns on the limiter, and one long that sets the limit

Reply:
Done.